### PR TITLE
Add python3 compatibility

### PR DESCRIPTION
**Pull request by mpgomez**
Add python3 compatibility:
 * Python3 compatibility
 * Update prospector and fix prospector errors

How this has been tested
I have run the tests (py.test) using python2.7 and python3. Tests and prospector happy locally and in CI :)
Thank you very much!

### CODE REVIEW

File: colander_tools/bytes.py [hunk 1]
```diff
@@ -46,5 +48,5 @@ class Base64Bytes(AbstractEncodedBytes):
 
 
 class URLSafeBase64Bytes(AbstractEncodedBytes):
-    encoder = staticmethod(base64.urlsafe_b64encode)
+    encoder = staticmethod(base64.standard_b64encode)
```
Comment (damz) [inline colander_tools/bytes.py]: That... looks odd?

File: requirements.txt [hunk 2]
```diff
@@ -1 +1,3 @@
 -e .
+colander
+pytz
```
Comment (damz) [inline requirements.txt]: Why do you need these if they are in `setup.py` already?

**Review by damz — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (mpgomez) [reply requirements.txt]: yup. That must go!

File: colander_tools/bytes.py (hunk 1 above)
Comment (mpgomez) [reply colander_tools/bytes.py]: that was very wrong

**Pull request merged**